.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
It seems throughout the JS and CSS you are using 2 spaces of indentation whereas here you are using 4. It should be uniform throughout, 2 seems to be the most common convention though this may vary depending on your workplace. Whichever you use it should be consistent throughout